### DISCUSSION TITLE
Changes to feed

### DISCUSSION THREAD

**Discussion by ric355** [community/community#66336] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

With the new feed, no matter what I do, I now can't see the submissions in other repos that I used to be able to see. That's a backwards step. All I can get are three recommended or trending repos that I'm not interested in at all.

Very disappointing.

**Comment by Luluno01** [comment]
Same. No way to figure out if someone has pushed to my repositories.

**Comment by roe-dl** [comment]
I experience the same. Work is much more difficult due to these changes.
